openapi.json

```diff
@@ -781,7 +781,17 @@
                 }
               }
             }
-          }
+          },
+          "internal_id": {
+            "type": "string",
+            "examples": "My ID",
+            "description": "The internal id given by the developer (push transfers only)",
```

Comment:
The internal ID if provided by the developer (push transfers only).

Reply:
The error's corresponding internal ID if provided by the developer (push transfers only).